data/en/japan2020/committee.yml

```diff
@@ -40,3 +40,9 @@ http://www.mushagaeshi.com
     img: "kusano.jpg"
     bio: "<p>2011年より富士通(株)に在籍し、データベース関連のデータ加工製品やネットワーク高速化ミドルウェアなどの製品開発、マーケティング、現場での適用支援など幅広い業務に従事。現在は富士通製Javaアプリケーションサーバである「Interstage」のマーケティングや適用支援を担当。最近はセミナーやブログでMicroProfileの啓蒙活動を行っている。</p>"
     anchor: "Kusano Yuichi"
+  - name: "Takashi Ito"
+    title: "Oracle"
+    twitter_handle: "itakash"
+    img: "TakashiIto.jpg"
+    bio: "<p>2010年、サン・マイクロシステムズ買収により日本オラクル株式会社に所属。Java、Java EEのコミュニティ活動支援に携わりつつ、Oracle WebLogic Server等のミドルウェア製品の販促を担当。2019年からOracle Corporationに移籍、WebLogic Server, Coherence、Heidonを開発する部門にて日本、およびアジアパシフィック地域を担当するProduct Managerとして活動中。</p>"
```

Comment:
Will you be translating this later, or was this a copy-paste error?

Reply:
will translate later.